.gitignore

```diff
@@ -8,3 +8,4 @@
 node/mocks
 build/reports
 /.run/
+*.json
```

Comment:
what is this for?

Reply:
i didn't add it to the gitignore. i'm not sure

Reply:
then please undo it

Reply:
this is for spell check

Reply:
I don't think we want to ignore all json files in our project. could you replace it with [this](https://github.com/IBM/ibm-block-csi-driver/blob/1ec0dd1e8030f75873c2bc19265520e88c8a2c22/.gitignore#L11)?

Reply:
i updated for the specific file that it affected .let me know if its a problem